### PR TITLE
Dev documentation

### PR DESCRIPTION
**Pull request by torr-c**

### CODE REVIEW

File: website/src/util/docs/index.md [hunk 1]
```diff
@@ -623,3 +623,332 @@ UnDistro and all required dependencies
...
+
+Both backend and frontend software lies on the same repository. The second is mostly composed by the _UI_ folder, while the first by the _api_, _charts_, _cmd_ and _tilt\_modules_ folders and also by many files in the root directory. Such folders will be discussed with more details later on.
+ 
+
+### Backend
+
+Undistro's backed is mostly written in the Go programming language. Go is a concise, statically typed, compiled language with a powerful concurrency model, also being the language in which Kubernetes is written. Most software repositories have a package for Go, but any case, it can also be obtained in [this page](https://golang.org/dl/).
+
+Another crucial piece of software to write code for Undistro is Helm, which is a component manager for Kubernetes, providing an unified way to share, track and upgrade cluster's setups. Helm is distributed as a single statically linked binary, which can be obtained [here](https://helm.sh/docs/intro/install/).
```
Comment (joivo) [inline website/src/util/docs/index.md:641]:
```suggestion
Another crucial piece of software to write code for Undistro is Helm, which is a package manager for Kubernetes, providing a unified way to share, track and upgrade cluster setups. Helm is distributed as a single statically linked binary, which can be obtained [here](https://helm.sh/docs/intro/install/).
```

File: website/src/util/docs/index.md [hunk 2]
```diff
@@ -623,3 +623,332 @@ UnDistro and all required dependencies
...
 &nbsp;
+
+
+
+# 11 - Development
+## How to setup the development environment
+
+The first step in getting involved with Undistro is to download the source code, and since the development is carried out by means of the Git version control system, it can be used to clone the [repository](https://github.com/getupio-undistro/undistro). If you don't already have Git installed, it can be found in most package managers as well as in the official downloads [page](https://git-scm.com/downloads). Furthermore, a Github account will also be required for sending changes back, so be sure to have one.
+
+Both backend and frontend software lies on the same repository. The second is mostly composed by the _UI_ folder, while the first by the _api_, _charts_, _cmd_ and _tilt\_modules_ folders and also by many files in the root directory. Such folders will be discussed with more details later on.
```
Comment (joivo) [inline website/src/util/docs/index.md:634]:
```suggestion
Both backend and frontend software lies on the same repository. The second is mostly composed by the _UI_ folder, while the first is by the _apis_, _controllers_, _pkg_,_cmd_ and _tilt\_modules_, and so on. Such folders will be discussed with more details later on.
```

Comment (joivo) [reply website/src/util/docs/index.md:634]: Here may be interesting to talk about the charts folder in a separate way. What u think?

File: website/src/util/docs/index.md [hunk 3]
```diff
@@ -623,3 +623,332 @@ UnDistro and all required dependencies
...
+
+KinD has the advantage of speed, being very quick to create and execute clusters. On the other hand, Minikube wins at extensibility, its multiple runtime options permit a finer-grained setup, and the plugins greatly simplify its configuration. Choose one of them to proceed with the setup.
+
+In a similar way, Docker is another one of the requirements. Docker is a container runtime used to package programs and their dependencies in an image, making it very portable and isolating its execution from the host operating system. Install it with your package manager or through Docker's get started [page](https://www.docker.com/get-started).
```
Comment (joivo) [inline website/src/util/docs/index.md:647]:
```suggestion
In a similar way, Docker is another one of the requirements to set up UnDistro locally. Docker is a container runtime used to package programs and their dependencies in an image, making it very portable and isolating its execution from the host operating system. Install it with your package manager or through Docker's get started [page](https://www.docker.com/get-started).
```

**Review by joivo — COMMENTED** [review]
juts some tips

**Review by felipeweb — APPROVED** [review]

**Pull request merged**